src/pal/inc/rt/palrt.h

```diff
@@ -1502,6 +1502,7 @@ typedef struct _DISPATCHER_CONTEXT {
     PRUNTIME_FUNCTION FunctionEntry;
     DWORD EstablisherFrame;
     DWORD TargetIp;
+    PCONTEXT CurrentRecord;
```

Comment:
@parjong you are right, my mistake. 
The only downside of your change is that we unnecessarily copy more than 1kB of memory of the context twice (once here and once in the ExceptionTracker::InitializeCurrentContextForCrawlFrame) when we really need just the non-volatile registers. 
Could you please create a new structure called e.g. NonVolatileRegisters for x86 and use it instead of the CurrentRecord? That way, we would copy just a few registers instead of the whole CONTEXT.